modules/core/src/graphics/Renderer.ts

```diff
@@ -23,7 +23,7 @@ export class Renderer {
     // create the pixi.js application
     this.app = new Application({
       view: props.canvas,
-      resolution: 2,
+      resolution: 1,
```

Comment:
reduced resolution for better performance. downside is text is blurry. may be possible to set resolution independently for text